ElfUtils/include/ElfUtils/LinuxMap.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#if defined(__linux)
```

Comment:
Wouldn't it be more common to have the `#ifndef ELF_UTILS_LINUX_MAP_H_` outside of `#if defined(__linux)`? Of course it makes no practical difference, but one always expects the include guards first.

Reply:
moved